src/code42cli/cmds/auditlogs.py

```diff
@@ -31,7 +29,14 @@
 AUDIT_LOGS_DEFAULT_HEADER["userName"] = "AffectedUser"
 AUDIT_LOGS_DEFAULT_HEADER["userId"] = "AffectedUserUID"
 
-
+begin_option = opt.begin_option(
```

Comment:
Seems like a possibility of re-using code, some parameterized method for creating these two options

Reply:
That's kind of how it was before, but it was only re-usable in a specific way. I'd rather have there be a tiny bit of duplication in each command that uses these options and have them be flexible enough to not have to rewrite everything if a new command needs a slightly different configuration.

Reply:
agree with tim, code should only be reused if its beleived that the functionality will _always_ stay the same, not just because it is _currently_ the same.